src/main/java/gift/auth/dto/KakaoTokens.java

```diff
@@ -0,0 +1,16 @@
+package gift.auth.dto;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public record KakaoTokens(
+
+        @JsonProperty("token_type") String tokenType,
+        @JsonProperty("access_token") String accessToken,
+        @JsonProperty("expires_in") Integer expiresIn,
+        @JsonProperty("refresh_token") String refreshToken,
+        @JsonProperty("refresh_token_expires_in") Integer refreshTokenExpiresIn,
+        @JsonProperty("scope") String scope
+) {
```

Comment:
```suggestion
@JsonNaming(PropertyNamingStrategies.SnakeCaseStrategy.class)
public record KakaoTokens(

        String tokenType,
        String accessToken,
        Integer expiresIn,
        String refreshToken,
        Integer refreshTokenExpiresIn,
        String scope
) {
}
```

모두 스네이크 케이스로 변환해야한다면 , JsonProperty를 각각 달아주는 것 대신 @JsonNaming(PropertyNamingStrategies.SnakeCaseStrategy.class)를 사용해볼 수 있을 것 같아요.

Reply:
오 이런 유용한게 있었군요 감사합니다!